From 2e3e466b3c63dfbc5dea13efe45d3b36668b32aa Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 9 Jan 2024 09:08:49 +0100
Subject: [PATCH 0843/1141] arm64: dts: st: add simple-framebuffer on
 stm32mp257f-ev1 board

Implement a node dedicated to the handling of a simple-framebuffer
through the boot stages.

Change-Id: I1f1f1e53e87781345025e0eab194c3f4244676e9
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/349891
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -29,6 +29,14 @@
 
 	chosen {
 		stdout-path = "serial0:115200n8";
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		framebuffer {
+			compatible = "simple-framebuffer";
+			status = "disabled";
+		};
 	};
 
 	clocks {
